src/terraform/providers/terraform-provider-avere/README.md

```diff
@@ -110,6 +110,7 @@ The following arguments are supported:
 * <a name="enable_support_uploads"></a>[enable_support_uploads](#enable_support_uploads) - (Optional) This setting defaults to 'false' and by setting to 'true' you agree to the [Privacy Policy](https://privacy.microsoft.com/en-us/privacystatement) of the Avere vFXT.  This enables support exactly as described in the [Enable Support Uploads documentation](https://docs.microsoft.com/en-us/azure/avere-vfxt/avere-vfxt-enable-support).  Avere vFXT for Azure can automatically upload support data about your cluster. These uploads let support staff provide the best possible customer service.
 * <a name="support_uploads_company_name"></a>[support_uploads_company_name](#support_uploads_company_name) - (Optional, but if specified required with `"enable_support_uploads"`) Specifies the customer name on support uploads to help help Avere support identify support packages.  This is only used with support uploads.
 * <a name="enable_rolling_trace_data"></a>[enable_rolling_trace_data](#enable_rolling_trace_data) - (Optional, but if specified required with `"enable_support_uploads"`) This setting defaults to 'false' and by setting to 'true' this setting adds the rolling trace option to the `"enable_support_uploads"` feature and is useful for troubleshooting performance issues.
+* <a name="rolling_trace_flag"></a>[rolling_trace_flag](#rolling_trace_flag) - (Optional, but if specified required with `"enable_support_uploads"` and `"enable_rolling_trace_data"`) This setting defaults to '0xef4001'.
```

Comment:
It might be worth explaining what the default setting does as the code is quite opaque.

Reply:
this is updated now